README.md

```diff
@@ -103,6 +103,96 @@ spec:
     }
 ```
 
+#### Configuring SriovNetwork with the RDMA CNI Plugin
+
+RDMA CNI enables Pod exclusive access to RDMA resources (and their associated hardware counters).
+To use RDMA CNI the following is required:
+
+**1. RDMA subsystem mode set to `exclusive`**
+
+```yaml
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetworkPoolConfig
+metadata:
+  name: rdma-workers
+  namespace: sriov-network-operator
+spec:
+  maxUnavailable: 1
+  rdmaMode: exclusive
+  nodeSelector:
+    matchLabels:
+      node-role.kubernetes.io/worker: ""
+```
+
+**2. SriovNetwork is using RDMA CNI plugin as a meta plugin**
+
+```yaml
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetwork
+metadata:
+  name: rdma-network
+  namespace: sriov-network-operator
+spec:
+  resourceName: rdma_shared_device
+  networkNamespace: default
+  ipam: |
+    {
+      "type": "host-local",
+      "subnet": "10.10.10.0/24",
+      "rangeStart": "10.10.10.171",
+      "rangeEnd": "10.10.10.181"
+    }
+  metaPluginsConfig: |
+    {
+      "type": "rdma"
+    }
+```
+
+This configuration:
+- Uses `metaPluginsConfig` to inject the RDMA CNI plugin
+- Allows pods using this network exclusive access to RDMA resources including their associated hardware counters
+- Requires the nodes to be configured with RDMA mode set to exclusive
+- Works with SR-IOV network policies that have `isRdma: true` specified
+
+**3. SriovNetworkNodePolicy with `spec.isRdma=true` is used**
+
+```yaml
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetworkNodePolicy
+metadata:
+  name: policy-1
+  namespace: sriov-network-operator
+spec:
+  deviceType: netdevice
+  nicSelector:
+    pfName: ["eno1"]
+  nodeSelector:
+    feature.node.kubernetes.io/network-sriov.capable: "true"
+  numVfs: 4
+  priority: 90
+  resourceName: rdma_shared_device
```

Comment:
nit: rdma_exclusive_device ? 

if you agree, need to adjust also the resourceName in SriovNetwork